lib/packages/docutils/rst.nim

```diff
@@ -43,13 +45,17 @@
 ##   + comments
 ## * inline markup
 ##   + *emphasis*, **strong emphasis**,
-##     ``inline literals``, hyperlink references, substitution references,
-##     standalone hyperlinks
+##     ``inline literals``, hyperlink references (including embedded URI),
+##     substitution references, standalone hyperlinks,
+##     internal links (inline and outline)
 ##   + \`interpreted text\` with roles ``:literal:``, ``:strong:``,
 ##     ``emphasis``, ``:sub:``/``:subscript:``, ``:sup:``/``:supscript:``
 ##     (see `RST roles list`_ for description).
+##   + inline internal targets
 ##
-## Additional features:
+## Additional features of rst.nim:
+##
+## .. _`Nim-specific features`:
```

Comment:
This should be above the line `Additional features of rst.nim:`, otherwise when you click on the link, you don't see the "headline"

Reply:
changed it here and in other places.